packages/core/src/utilities/convertVolumeToStackViewport.ts

```diff
@@ -0,0 +1,99 @@
+import * as Types from '../types';
+import cache, { ImageVolume } from '../cache';
+import { ViewportType } from '../enums';
+import uuidv4 from './uuidv4';
+
+/**
+ * Converts a volume viewport to a stack viewport.
+ *
+ * @param params - The parameters for the conversion.
+ * @param params.viewport - The volume viewport to convert.
+ * @param params.options - The conversion options.
+ * @param [params.options.viewportId] - The new stackViewportId, If not provided, the volume viewport id will be used.
+ * @param [params.options.background] - The background color of the stack viewport.
+ * @param [params.options.decache] - Whether to decache the volume. Defaults to false.
+ *
+ * @returns The converted stack viewport.
+ */
+async function convertVolumeToStackViewport({
```

Comment:
Should this throw if the viewport is showing a non-acquisition plane modality?

Reply:
Hmmm, I don't think so, since it is only a `view` and does not matter regarding the underlying images